facebook-commerce.php

```diff
@@ -2389,121 +2297,6 @@ private function sync_facebook_products_using_background_processor() {
 		return true;
 	}
 
-
-	/**
-	 * Special function to run all visible products by uploading feed.
-	 *
-	 * @internal
-	 */
-	public function ajax_sync_all_fb_products_using_feed() {
```

Comment:
There's a reference in `WC_Facebook_Integration_Test` to this function, worth removing even if it's a test

Reply:
Done.